.vscode/launch.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug express",
+            "type": "node",
+            "request": "launch",
+            "cwd": "${workspaceFolder}/server",
+            "runtimeExecutable": "nodemon",
+            "program": "src/app.ts",
+            "restart": true,
+            "outputCapture": "std",
+            "sourceMaps": true,
+            "env": {
+                "NODE_ENV": "developement"
+            }
+        },
+        {
+            "name": "Debug React",
+            "type": "chrome",
+            "request": "launch",
+            "sourceMaps": true,
+            "smartStep": true,
+            "url": "http://localhost:5173",
+            "webRoot": "${workspaceFolder}/client",
+            "env": {
+                "NODE_ENV": "developement"
```

Comment:
There is a typo in the NODE_ENV value.
It should be **development** instead of **developement**.

Reply:
nice catch @NetanelSegal